Salary App/Base.lproj/Main.storyboard

```diff
@@ -44,30 +44,59 @@
                                     <action selector="submitBtnPressed:" destination="BYZ-38-t0r" eventType="touchUpInside" id="TPH-sh-CCE"/>
                                 </connections>
                             </button>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="28" translatesAutoresizingMaskIntoConstraints="NO" id="MvT-zZ-4Lz">
```

Comment:
Not many companies use storyboard find some resources on setting this up programatically